fuzzers/index.ts

```diff
@@ -29,10 +29,26 @@
 // * [Twitch](https://www.twitch.tv/gamozo)
 //
 
-import CorpusBasedFuzzer from 'corpus_based_fuzzer';
+import CorpusBasedFuzzer, { ICorpusLoader } from 'corpus_based_fuzzer';
 import Engine from 'engine';
-import ISO_CORPUS from 'iso_corpus';
+import isoCorpusLoader from 'iso_corpus';
+import xpath31FunctionCatalogCorpusLoader from 'xpath-31-function-catalog-corpus';
 
-const fuzzer = new CorpusBasedFuzzer(ISO_CORPUS);
-const engine = new Engine<CorpusBasedFuzzer>();
-engine.run(fuzzer, __filename);
+// Select the corpus to use
+const corpora: { [key: string]: ICorpusLoader } = {
+	[isoCorpusLoader.name]: isoCorpusLoader,
+	[xpath31FunctionCatalogCorpusLoader.name]: xpath31FunctionCatalogCorpusLoader
+};
+const corpusName = process.argv.slice(2)[0];
+const corpusLoader = corpora[corpusName];
+if (corpusLoader != null) {
```

Comment:
Missed a `!==` here.

Reply:
:barf: I don't know why my linter is still fubar, sad Js makes me sad. Will fix.

Reply:
Needs to be fixed before integration